src/kokkos/plugin-PixelVertexFinding/kokkos/gpuVertexFinder.cc

```diff
@@ -130,8 +130,8 @@ namespace KOKKOS_NAMESPACE {
           Kokkos::RangePolicy<KokkosExecSpace>(execSpace, 0, TkSoA::stride()),
           KOKKOS_LAMBDA(const size_t i) { loadTracks(tksoa, vertices_d, workspace_d, ptMin, i); });
 
-#ifdef KOKKOS_BACKEND_SERIAL
-      auto policy = TeamPolicy(execSpace, 1, 1).set_scratch_size(0, Kokkos::PerTeam(8192 * 4));
+#if defined KOKKOS_BACKEND_SERIAL || defined KOKKOS_BACKEND_PTHREAD
+      auto policy = TeamPolicy(execSpace, 1, Kokkos::AUTO()).set_scratch_size(0, Kokkos::PerTeam(8192 * 4));
```

Comment:
@PointKernel Could you double-check my changes in this file?

Reply:
I don't see any issues here.